backend/Code/FrequencyScores.py

```diff
@@ -0,0 +1,128 @@
+from bs4 import BeautifulSoup
+import requests
+from dicttoxml import dicttoxml
+from xml.dom.minidom import parseString
+import xmltodict
+import os
+
+# Class that retrieves the score of a letter in a specified language according
+# to its frequency
+
+# Base url of the requests
+BASE_URL = "http://www.sttmedia.com/"
+CHARS = "characterfrequency"
+SYLLABLES = "syllablefrequency"
+
+# Select the min and max score values possible for any letter
+DEF_MIN = 1
+DEF_MAX = 20
+
+# Language from which to retrieve the letter scores (as it appears
+# in the web page)
+LANG = "nederlands"
+FOLDER = LANG + "Scores"
+
+# This is the main function for making queries.
```

Comment:
Add the comments inside the function not on the outside. ALSO! Make the private function private and i _suggest_ that you use [this convention](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings) for commenting in _in_ classes. One of the problems in Python is that you don't know what objects you are handling since - in contrast to Java or C/C++ - you don't have to declare the type. For that reason _some_ developers prefer that convention where you specify what is the type of each variable in the comment.